string alarm_name             # Name of the alarm.
bool raised                   # Do we want to raise or clear the alarm?

string node_name              # Name of the node raising/clearing the alarm.
bool action_required          # Does something need to happen?
string problem_description    # A brief human-readable description of the alarm.
string parameters             # A JSON problem description.
uint8 severity                # How severe is the alarm? 0 is the most severe.
---
bool succeed
